mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.js

```diff
@@ -180,6 +187,7 @@ CheckboxGroup.propTypes = {
 
 CheckboxGroup.defaultProps = {
   onBlur: () => {},
+  optionsExcludedFromShowingSelectedMarkup: [],
```

Comment:
This is really weird, the default prop isnt working and Im puzzled as to why. The composing component isnt setting this prop, so it should be an empty array....